.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,29 +1,12 @@
 # Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
 blank_issues_enabled: false  # default: true
 contact_links:
-- name: 🔐 Security bug report 🔥
-  url: https://github.com/aio-libs/.github/security/policy
-  about: |
-    Please learn how to report security vulnerabilities here.
-
-    For all security related bugs, send an email
-    instead of using this issue tracker and you
-    will receive a prompt response.
-
-    For more information, see
-    https://github.com/aio-libs/.github/security/policy
 - name: 🤷💻🤦 StackOverflow
   url: https://stackoverflow.com/questions/tagged/aiohttp
   about: Please ask typical Q&A here
-- name: 💬 Gitter Chat
-  url: https://gitter.im/aio-libs/Lobby
-  about: Chat with devs and community
 - name: 💬 Discourse
   url: https://aio-libs.discourse.group/
   about: Please start usage discussions here
-- name: 💬 Mailing list
-  url: https://groups.google.com/forum/#!forum/aio-libs
-  about: Usage Q&A
-- name: 📝 Code of Conduct
```

Comment:
This one is an important reminder, being visible like this it calms down people who are about to make angry postings... Also, it's important for community health because when there's a CoC in a visible place, it encourages people to consider contributing.

Reply:
Put the reference in README then.
Issue templates are for *filing an issue* or *asking a question*, not for links to informational pages (despite how these pages are important).

Reply:
The problem is that people would want to file an issue about these. And we could intercept these attempts right there, when they click "New issue" they'll see that they shouldn't do this. Do you honestly think that everyone reads the docs? We both know that many people use issues tracker instead of google...

Reply:
If people want they just file the issue using the first available template, we both know it.
Blowing a list of available options makes more mess than help IMHO.

Reply:
Okay, do as you will